MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +291,32 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        guard mapView.tracksUserCourse else { return } //only adjust when we are actively tracking user course
+        
+        let zoomOutAltitude = RouteMapViewController.zoomedOutManeuverAltitude
+        let defaultAltitude = mapView.defaultAltitude
+        let isLongRoad = routeProgress.distanceRemaining >= RouteMapViewController.longManeuverDistance
+        
+        let isMotorway : (RouteStep?) -> Bool? = { $0?.intersections?.first?.outletRoadClasses?.contains(.motorway) }
```

Comment:
We have a `isMotorway` motorway function. I'd say, maybe move it to an extension on `RouteStep`.